warehouse/stats.go

```diff
@@ -257,12 +206,41 @@ func (job *UploadJob) recordLoadFileGenerationTimeStat(startID, endID int64) (er
 	return nil
 }
 
+func (jr *jobRun) defaultTags() stats.Tags {
+	return stats.Tags{
+		"module":      moduleName,
+		"destType":    jr.job.DestinationType,
+		"warehouseID": warehouseTagName(jr.job.DestinationID, jr.job.SourceName, jr.job.DestinationName, jr.job.SourceID),
+		"workspaceId": jr.job.WorkspaceID,
+		"destID":      jr.job.DestinationID,
+		"sourceID":    jr.job.SourceID,
+	}
+}
+
+func (jr *jobRun) timerStat(name string, extraTags ...warehouseutils.Tag) stats.Measurement {
+	tags := jr.defaultTags()
```

Comment:
[optional] `defaultTags` could be `buildTags` and get the `extraTags` as well to avoid the repetition of the `for` loop as well.